pkg/datastore/sql_struct.go

```diff
@@ -30,48 +30,52 @@ import (
 
 const (
 	// Struct Field Names.
-	FIELD_ORGID = "OrgId"
+	FIELD_ORGID      = "OrgId"
+	FIELD_INSTANCEID = "InstanceId"
 
 	// SQL Columns.
-	COLUMN_ORGID    = "org_id"
-	COLUMN_REVISION = "revision"
+	COLUMN_ORGID      = "org_id"
+	COLUMN_INSTANCEID = "instance_id"
+	COLUMN_REVISION   = "revision"
 
 	// Messages.
 	REVISION_OUTDATED_MSG = "Invalid update - outdated "
 )
 
-// Maps a DB table name to booleans showing if the table is multitenant.
-var multitenancyMap = make(map[string]bool)
+// Maps DB table name to booleans showing if the table is multitenant and revisioning.
```

Comment:
**and supports revisioning**

Reply:
You should mention the keys for the internal map (COLUMN_REVISION, COLUMN_ORGID, COLUMN_INSTANCEID)